### PR TITLE
Adds a sea turtle.

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

Art by UbaserB

Adds a sea turtle
Adds a pet sea turtle who cant slip for janis maybe named Squirt.
Adds turtle eggs.
Adds a sea turtle crate

image
image

## Why / Balance

There weren't enough turtles, ya know?

Also, more zookeeper content real.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Ubaser, IProduceWidgets
- add: Sea turtles

### CODE REVIEW

**Comment by UbaserB** [comment]
could you put me in the CL so i can track which PRs i contributed to?

**Comment by UbaserB** [comment]
AUGH you did it wrong, if you put my name your name wont show up, its gotta be “IProduceWidgets, Ubaser”

**Comment by lzk228** [comment]
sprite us kinda off

**Comment by UbaserB** [comment]
Oh and the CL description should also be proper.. like uhhh."Sea turtles can now be bought from cargo, and the janitor starts with a pet named Squirt" or something. And lzk228 uhhhh uh uh turtle are HARD to draw you know!! lol.

**Comment by IProduceWidgets** [comment]
Well I didn't map them anywhere yet. Usually they Ive found its preferred to get the entities added before they get mapped.

**Comment by IProduceWidgets** [comment]
> sprite is kinda off

Is it the flat top on lighter tiles? I went back and forth on that a touch. Looks definitely good on dark tiles to me. Might just be because Im looking too closely though.

File: Resources/Locale/en-US/prototypes/catalog/fills/crates/livestock-crates.ftl [hunk 1]
```diff
@@ -53,4 +53,7 @@ ent-CrateNPCLizard = Lizard crate
     .desc = A crate containing a lizard.
 
 ent-CrateNPCKangaroo = Kangaroo crate
-    .desc = A crate containing a kangaroo.
\ No newline at end of file
+    .desc = A crate containing a kangaroo.
+
+ent-CrateNPCTurtle = Sea turtle crate
+    .desc = A crate containing a Sea turtle.
```
Comment (lzk228) [inline Resources/Locale/en-US/prototypes/catalog/fills/crates/livestock-crates.ftl:59]:
```suggestion
    .desc = A crate containing a sea turtle.
```

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 2]
```diff
@@ -2711,3 +2711,85 @@
   - type: NpcFactionMember
     factions:
     - Passive
+
+- type: entity
+  name: turtle 
+  parent: SimpleMobBase
+  id: MobTurtle #Just trust me it does not make sound. https://www.npr.org/2022/10/31/1132951238/what-sound-does-a-turtle-make
+  description: splish splash in the bath
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:2719]:
```suggestion
  description: Splish splash in the bath.
```

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 3]
```diff
@@ -722,3 +722,26 @@
     attributes:
       proper: true
       gender: male
+
+- type: entity
+  name: Squirt
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:727]: squirt 😳😳😳

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 4]
```diff
@@ -722,3 +722,26 @@
...
+  parent: MobTurtle
+  id: MobTurtleSquirt
+  description: She's a long way from the ocean...
+  components:
+  - type: InteractionPopup
+    successChance: 0.6
+    interactSuccessString: petting-success-kangaroo
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:734]: kangaroo?

**Review by lzk228 — CHANGES_REQUESTED** [review]
also add spawner of turtle pet

and it would be cool to add turtle's shell, after butching in can be weared as helmet

**Comment by lzk228** [comment]
and maybe add a tortoise instead? at least tortoises is more land animals, and on station is no so much water for sea turtle

**Comment by UbaserB** [comment]
> also add spawner of turtle pet
> 
> and it would be cool to add turtle's shell, after butching in can be weared as helmet

i actually made a sprite of it earlier before scratching the idea. However, not as a helmet but for a shield instead.

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml (hunk 4 above)
Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:734]: ehh, we've already got Paperwork whos a slow station pet. Also we have the water entity, so we could use at least one mob for that, and I think this is at least better than a fish. (right now all we have is crab.)

**Comment by IProduceWidgets** [comment]
> and maybe add a tortoise instead? at least tortoises is more land animals, and on station is no so much water for sea turtle

Ehh, we've already got Paperwork whos a slow station pet. Also we have the water entity, so we could use at least one mob for that, and I think this is at least better than a fish. (right now all we have is crab.)

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml (hunk 3 above)
Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:727]: I'm not saying I stole it from Finding Nemo, but I am aware that is a turtles name in Finding Nemo.

**Comment by IProduceWidgets** [comment]
> > and it would be cool to add turtle's shell, after butching in can be weared as helmet
> 
> i actually made a sprite of it earlier before scratching the idea. However, not as a helmet but for a shield instead.

I think it would make a funny armor slot.

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 2 above)
Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: Resolved lies because rebase, I did it.

**Comment by IProduceWidgets** [comment]
Ehhh, eats should say kills, they don't consume them.

**Comment by IProduceWidgets** [comment]
Alright I think its feature complete, as far as I can think of.

**Comment by IamVelcroboy** [comment]
> Alright I think its feature complete, as far as I can think of.

Wait wait, one more request... Pouring mutagen on it causes it to grow to be human size and possess remarkable martial arts skills.

**Comment by UbaserB** [comment]
> > Alright I think its feature complete, as far as I can think of.
> 
> Wait wait, one more request... Pouring mutagen on it causes it to grow to be human size and possess remarkable martial arts skills.

cursed

**Comment by IamVelcroboy** [comment]
> > > Alright I think its feature complete, as far as I can think of.
> > 
> > 
> > Wait wait, one more request... Pouring mutagen on it causes it to grow to be human size and possess remarkable martial arts skills.
> 
> cursed

I think you mean, brilliant!

**Comment by IProduceWidgets** [comment]
> > Alright I think its feature complete, as far as I can think of.
> 
> Wait wait, one more request... Pouring mutagen on it causes it to grow to be human size and possess remarkable martial arts skills.

Would require a new system for mutagen. Also probably makes more sense as a change for all animals/mobs not just turtles. Beyond scope!

**Comment by IProduceWidgets** [comment]
Merge conflict fixed. I think it was just mad about a rebase.

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 5]
```diff
@@ -2749,3 +2749,111 @@
...
+  name: sea turtle 
+  parent: SimpleMobBase
+  id: MobTurtle
+  description: Splish splash in the bath.
+  components:
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 2
+    baseSprintSpeed: 3
+  - type: Sprite
+    drawdepth: Mobs
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:2762]: unnecessary

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 6]
```diff
@@ -2749,3 +2749,111 @@
...
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: turtle
+      sprite: Mobs/Animals/turtle.rsi
+  - type: Physics
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:2767]: unnecessary

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 7]
```diff
@@ -2749,3 +2749,111 @@
...
+    successChance: 0.3
+    interactSuccessString: petting-success-reptile
+    interactFailureString: petting-failure-generic
+    interactSuccessSpawn: EffectHearts
+    #interactSuccessSound:
+    #  path: /Audio/Animals/lizard_happy.ogg #Just trust me it does not make sound. https://www.npr.org/2022/10/31/1132951238/what-sound-does-a-turtle-make
+  - type: Bloodstream
+    bloodMaxVolume: 150
+  - type: Damageable
+    damageContainer: Biological
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:2826]: unnecessary

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 8]
```diff
@@ -2749,3 +2749,111 @@
...
+    - id: FoodMeatLizard
+      amount: 1
+  - type: InteractionPopup
+    successChance: 0.3
+    interactSuccessString: petting-success-reptile
+    interactFailureString: petting-failure-generic
+    interactSuccessSpawn: EffectHearts
+    #interactSuccessSound:
+    #  path: /Audio/Animals/lizard_happy.ogg #Just trust me it does not make sound. https://www.npr.org/2022/10/31/1132951238/what-sound-does-a-turtle-make
+  - type: Bloodstream
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:2823]: unnecessary

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 9]
```diff
@@ -722,3 +722,26 @@
...
       gender: male
+
+- type: entity
+  name: Squirt
+  parent: MobTurtle
+  id: MobTurtleSquirt
+  description: She's a long way from the ocean...
+  components:
+  - type: InteractionPopup
+    successChance: 0.6
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:733]: if you want to rewrite, you can left only this line

**Review by lzk228 — COMMENTED** [review]
and i still don't like a sprite

File: Resources/Textures/Mobs/Animals/turtle.rsi/meta.json [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+          1
+        ]
+      ]
+    },
+    {
+      "name": "turtle",
+      "directions": 4,
+      "delays": [
+        [
+          1
```
Comment (TheShuEd) [inline Resources/Textures/Mobs/Animals/turtle.rsi/meta.json:23]: no need to prescribe animation delays if we have only 1 frame

**Comment by TheShuEd** [comment]
are you going to finish this PR?

**Comment by IProduceWidgets** [comment]
Yes.

**Comment by IProduceWidgets** [comment]
Sorry for the million review requests, I uh, done goofed.

**Comment by metalgearsloth** [comment]
Same here https://github.com/space-wizards/space-station-14/pull/21553#issuecomment-1913927013

**Comment by AJCM-git** [comment]
New fluff animals ordered from cargo or spawned in the station have been added to the freeze list #8524 with new requirements for they to be approved (tldr is that they must have something cool going on apart from being cute and having sprites, like new AI behaviors or being spawned in proc gens maps), feel free to re-open if you meet them.

**Pull request closed without merging**